packages/storybook/stories/va-minimal-footer.stories.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { getWebComponentDocs, propStructure, StoryDocs } from './wc-helpers';
+
+const minimalFooterDocs = getWebComponentDocs('va-minimal-footer');
+
+export default {
+  title: 'Components/Minimal Footer',
+  id: 'components/va-minimal-footer',
+  parameters: {
+    componentSubtitle: `va-minimal-footer web component`,
+    docs: {
+      page: () => <StoryDocs data={minimalFooterDocs} />,
+    },
+  },
+};
+
+const defaultArgs = {};
+
+const Template = ({ }) => {
```

Comment:
You can probably just do 🤷‍♀️:

```
const defaultArgs = {};

const Template = (defaultArgs) => {
```
Or remove `defaultArgs` completely. I think the example I linked to before was done a little strange: